packages/fxa-settings/src/pages/Index/en.ftl

```diff
@@ -0,0 +1,13 @@
+# Index / home page
```

Comment:
```suggestion
## Index / home page
```

Non-blocking, but this might be useful context so suggest making this a group comment.

Reply:
Thanks! That's what I meant to do, the single vs double gets me sometimes since I'm used to Markdown. 😆